apps/framework-docs/src/pages/moose/olap/model-table.mdx

```diff
@@ -612,6 +612,23 @@ s3_events = OlapTable[S3Event]("s3_events", OlapConfig(
 S3Queue requires ClickHouse 24.7+ and proper ZooKeeper/ClickHouse Keeper configuration for coordination between replicas. Files are processed exactly once across all replicas.
 </Callout>
 
+<Python>
```

Comment:
is there any similar behavior for TS @phiSgr ?

Reply:
Not really. Only quotation marks needed.